datasource/src/main/resources/liquibase/changelog/changelog-0.1.xml

```diff
@@ -32,4 +32,13 @@
         </rollback>
     </changeSet>
 
+    <changeSet id="add_column_for_user_deactivation" author="ysheiko">
+        <addColumn tableName="user">
+            <column name="active" type="${boolean.type}" defaultValueBoolean="true"/>
```

Comment:
Please, add `NotNullConstraint` constraint here. I almost forgot about this one.

Reply:
It used to be here, brought it back